vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -97,32 +99,39 @@ switch -Regex ($windowsSku) {
 
 $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes/pause:3.9-hotfix-20230808",
+
     # This is for test purpose only to reduce the test duration.
     "mcr.microsoft.com/windows/servercore/iis:latest",
+
     # CSI. Owner: andyzhangx (Andy Zhang)
     "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.12.0", # for k8s 1.27+
     "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.13.1", # for k8s 1.30+
     "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.14.0", # for k8s 1.30+
+
     "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.10.1", # for k8s 1.27+
     "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.11.1", # for k8s 1.30+
     "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.12.0", # for k8s 1.30+
+
     "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.28.10-windows-hp", # for k8s 1.27.x
```

Comment:
It seems like that we can remove azuredisk-csi:v1.28.10-windows-hp, azuredisk-csi:v1.29.9-windows-hp, azuredisk-csi:v1.30.4-windows-hp, azurefile-csi:v1.28.12-windows-hp, azurefile-csi:v1.29.8-windows-hp, azurefile-csi:v1.30.5-windows-hp, azure-cns:v1.5.3, azure-cns:v1.6.7

Reply:
Will remove. I assume the azure-cns:v1.5.3 is actually 1.5.35 (as .38 is also there).